### ISSUE TITLE
FamixTag color serialization

### ISSUE THREAD

**Issue by sbragagnolo** [moosetechnology/Famix#368]
Famix tag serialize colors as hexadecimal. As far as i know this is due to the transmition of color to a web front end, where colors in hexadecimal make sense. 
Still, this make harder the inspection of the objects in pharo. We should transfer the concern of serialization to a serializer.

**Comment by jecisc** [comment]
The historic reason is that Fame handles only:
- Strings
- Numbers
- Booleans
- Fame objects

It is not possible to have a colour as the value of a property. Thus we stored the colour as a string and the hexadecimal value was an easy way to store it.

**Comment by sbragagnolo** [comment]
Yeah. I saw that with other objects. I am now even more sure that we need to be able to articulate secondary serializes for other kind of objects! :)
